internal/errorsx/errorsx_test.go

```diff
@@ -74,7 +74,7 @@ func TestToFailureString(t *testing.T) {
 		}
 	})
 	t.Run("for system timeout", func(t *testing.T) {
-		if toFailureString(syscall.ETIMEDOUT) != FailureGenericTimeoutError {
+		if toFailureString(syscall.ETIMEDOUT) != FailureTimedOut {
```

Comment:
This mapping does not hide information. (There are too many `generic_timeout_error` cases already.)